docs/README.md

```diff
@@ -104,32 +124,32 @@ COR_ENABLE_PROFILING=1
 COR_PROFILER={918728DD-259F-4A6A-AC2B-B85E1B658318}
 CORECLR_ENABLE_PROFILING=1
 CORECLR_PROFILER={918728DD-259F-4A6A-AC2B-B85E1B658318}
-DOTNET_ADDITIONAL_DEPS=%InstallationLocation%/AdditionalDeps
-DOTNET_SHARED_STORE=%InstallationLocation%/store
-DOTNET_STARTUP_HOOKS=%InstallationLocation%/netcoreapp3.1/OpenTelemetry.AutoInstrumentation.StartupHook.dll
-OTEL_DOTNET_AUTO_HOME=%InstallationLocation%
-OTEL_DOTNET_AUTO_INTEGRATIONS_FILE=%InstallationLocation%/integrations.json
+DOTNET_ADDITIONAL_DEPS=$INSTALL_DIR/AdditionalDeps
+DOTNET_SHARED_STORE=$INSTALL_DIR/store
+DOTNET_STARTUP_HOOKS=$INSTALL_DIR/netcoreapp3.1/OpenTelemetry.AutoInstrumentation.StartupHook.dll
+OTEL_DOTNET_AUTO_HOME=$INSTALL_DIR
+OTEL_DOTNET_AUTO_INTEGRATIONS_FILE=$INSTALL_DIR/integrations.json
 ```
 
 On **Windows** you need to additionally set:
 
 ```env
-COR_PROFILER_PATH_64=%InstallationLocation%/win-x64/OpenTelemetry.AutoInstrumentation.Native.dll
-COR_PROFILER_PATH_32=%InstallationLocation%/win-x86/OpenTelemetry.AutoInstrumentation.Native.dll
-CORECLR_PROFILER_PATH_64=%InstallationLocation%/win-x64/OpenTelemetry.AutoInstrumentation.Native.dll
-CORECLR_PROFILER_PATH_32=%InstallationLocation%/win-x86/OpenTelemetry.AutoInstrumentation.Native.dll
+COR_PROFILER_PATH_64=$INSTALL_DIR/win-x64/OpenTelemetry.AutoInstrumentation.Native.dll
```

Comment:
On Windows this is not the correct way to expand an env var.

Reply:
What do you mean by "in Windows"? E.g. in PowerShell it is the correct way 😉